Sources/Sentry/SentryProfiler.mm

```diff
@@ -50,6 +50,30 @@
 #        import <UIKit/UIKit.h>
 #    endif // SENTRY_HAS_UIKIT
 
+#    if defined(TEST) || defined(TESTCI)
```

Comment:
`m`: As this code is only required for tests, wouldn't it make sense to put that into a file in the `SentryTests/` folder?

Reply:
This will be tricky. It's only used in the UI tests, not the unit tests, so exposing internal implementation from the SDK isn't as straightforward. If we bring in SentryProfiler, then we have to bring in every other import as well.

Let me see if just putting an ObjC category in the app can still be guaranteed to run its `+[load]` method after our ObjC classes in the SDK.

Reply:
Again, this is acceptable. Please add a comment to explain this. Isolating this in a category is a great idea.

Reply:
Will be done in a later PR